task/buildpacks-separate-phases/0.1/README.md

```diff
@@ -1,35 +1,28 @@
 # Cloud Native Buildpacks
 
 This build template builds source into a container image using [Cloud Native
-Buildpacks](https://buildpacks.io).
+Buildpacks](https://buildpacks.io). To do that, it uses [builders](https://buildpacks.io/docs/concepts/components/builder/#what-is-a-builder) to run buildpacks against your application.
 
-The Cloud Native Buildpacks website describes v3 buildpacks as:
+Cloud Native Buildpacks are pluggable, modular tools that transform application source code into OCI images. They replace Dockerfiles in the app development lifecycle, and enable for swift rebasing of images and modular control over images (through the use of builders), among other benefits. This command uses a builder to construct the image, and pushes it to the registry provided.
 
-> ... pluggable, modular tools that translate source code into container-ready
-> artifacts such as OCI images. They replace Dockerfiles in the app development
-> lifecycle with a higher level of abstraction. ...  Cloud Native Buildpacks
-> embrace modern container standards, such as the OCI image format. They take
-> advantage of the latest capabilities of these standards, such as remote image
-> layer rebasing on Docker API v2 registries.
+See also [`buildpacks`](../buildpacks) for the combined version of this task, which uses the [creator binary](https://github.com/buildpacks/spec/blob/platform/0.3/platform.md#operations), to run all of the [lifecycle phases](https://buildpacks.io/docs/concepts/components/lifecycle/#phases). This task, in contrast, runs all of the phases separately.
 
 ## Install the Task
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/buildpacks-v3/0.1/buildpacks-v3.yaml
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/task/buildpacks-separate-phases/0.1/buildpacks-separate-phases.yaml
```

Comment:
In an attempt to minimize the name, would `buildpacks-phases.yaml` be sufficient to describe the difference without the full name?